package.json

```diff
@@ -38,7 +38,7 @@
     "Tim Sullivan <tim@elastic.co>"
   ],
   "scripts": {
-    "test": "grunt test; grunt test:visualRegression",
+    "test": "bash -c 'grunt test $@; grunt test:visualRegression $@' --",
```

Comment:
This does mean development has to be done on a system with `bash` on it, which may be too restrictive.